conda: error: unrecognized arguments: --force

[conda does not support --force option in July 2023 in conda 23.7.0](https://github.com/conda/conda/issues/13704). So, if you are encounting this issues with a newer conda. Please replace two `--force` to `--yes` in file of `continuousflex/__init__.py`

After replacement, I encountered this error when I'm installing.

gcc -pthread -B /home/cryosparc_user/scipion/software/em/ContinuousFlex-3.4.1/compiler_compat -Wl,--sysroot=/ -Wsign-compare -fwrapv -Wall -O3 -DNDEBUG -fPIC -DBOOST_ALL_NO_LIB=1 -DBOOST_THREAD_BUILD_DLL=1 -DBOOST_MULTI_INDEX_DISABLE_SERIALIZATION=1 -DBOOST_PYTHON_SOURCE=1 -Dboost=pycudaboost -DBOOST_THREAD_DONT_USE_CHRONO=1 -DPYGPU_PACKAGE=pycuda -DPYGPU_PYCUDA=1 -DHAVE_CURAND=1 -Isrc/cpp -Ibpl-subset/bpl_subset -I/home/cryosparc_user/scipion/software/em/ContinuousFlex-3.4.1/lib/python3.8/site-packages/numpy/core/include -I/home/cryosparc_user/scipion/software/em/ContinuousFlex-3.4.1/include/python3.8 -c src/cpp/cuda.cpp -o build/temp.linux-x86_64-3.8/src/cpp/cuda.o
      In file included from src/cpp/cuda.cpp:4:
      src/cpp/cuda.hpp:14:10: fatal error: cuda.h: No such file or directory
         14 | #include <cuda.h>
            |          ^~~~~~~~
      compilation terminated.
      error: command 'gcc' failed with exit status 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pycuda
ERROR: Could not build wheels for pycuda, which is required to install pyproject.toml-based projects

failed

CondaEnvException: Pip failed

ERROR: File or folder '/home/cryosparc_user/scipion/software/em/ContinuousFlex-3.4.1/last-pull-3.4.1.txt' not found after running 'eval "$(/home/cryosparc_user/miniconda3/bin/conda shell.bash hook)"&&conda env create -f /home/cryosparc_user/scipion-em-continuousflex/continuousflex/conda_noCuda.yaml --prefix . --yes&& conda env update --prefix . --file=/home/cryosparc_user/scipion-em-continuousflex/continuousflex/conda.yaml && touch last-pull-3.4.1.txt'.

--- a/continuousflex/__init__.py
+++ b/continuousflex/__init__.py
@@ -110,7 +110,7 @@ class Plugin(pwem.Plugin):
         def getCondaInstallation(version, txtfile):
             installationCmd = cls.getCondaActivationCmd()
             config_path = continuousflex.__path__[0] + '/conda_noCuda.yaml'
-            installationCmd += 'conda env create -f {} --prefix . --force'.format(config_path)
+            installationCmd += 'conda env create -f {} --prefix . --yes'.format(config_path)
             # If nvcc is in the path, install Optical Flow and DeepLearning Libraries
             if os.popen('which nvcc').read() is not None:
                 config_path = continuousflex.__path__[0] + '/conda.yaml'

